lib/godwoken_explorer_web/controllers/api/account_controller.ex

```diff
@@ -3,17 +3,49 @@ defmodule GodwokenExplorerWeb.API.AccountController do
 
   alias GodwokenExplorer.{Repo, Account}
 
-  def show(conn, params) do
-    result = case Repo.get(Account, params["id"]) do
-      %Account{} -> Account.find_by_id(params["id"])
-      nil ->  %{
-        error_code: 404,
-        message: "not found"
-      }
-    end
+  def show(conn, %{"id" => id}) do
+    result =
+      case Repo.get(Account, id) do
+        %Account{} ->
+          Account.find_by_id(id)
+          |> account_to_view()
+
+        nil ->
+          %{
+            error_code: 404,
+            message: "not found"
+          }
+      end
+
     json(
       conn,
       result
     )
   end
+
+  defp account_to_view(account) do
+    account = %{account | ckb: balance_to_view(account.ckb, 8)}
+
+    account =
+      with udt_list when not is_nil(udt_list) <- Kernel.get_in(account, [:user, :udt_list]) do
```

Comment:
```suggestion
      with udt_list when udt_list != [] <- Kernel.get_in(account, [:user, :udt_list]) do
```

Reply:
`udt_list` would be `nil` when `type` is neither `user` nor `sudt`.